config/mcus/manufacturer/Fysetc_ERCF_ERB.cfg

```diff
@@ -0,0 +1,24 @@
+[board_pins mcu_ercf]
```

Comment:
Can you change the name of the MCU in the manufacturer file to something else ? I'm not sure it will work as is as this could result in a conflict if for some reason two pin share the same name between my own pin notation and what is written on the PCB (ie. something like SERVO=SERVO would cause trouble).

So I would prefer to change it here to something like `ercf_manufacturer` or whatever you think is better.